common/stats/status_server.cpp

```diff
@@ -98,12 +101,33 @@ StatusServer::StatusServer(uint16_t port, EndPointToOPMap op_map)
 }
 
 std::string StatusServer::GetPageContent(const std::string& end_point) {
-  auto itor = op_map_.find(end_point);
-  if (itor == op_map_.end()) {
-    return "Unsupported http path.\n";
+  // Add dummy url to allow it to be parsed by folly:Uri.
+  folly::Uri u("http://blah.blah" + end_point);
+  auto params = u.getQueryParams();
```

Comment:
I forgot one thing.
accoding to https://github.com/facebook/folly/blob/master/folly/Uri.h#L96-L98

getQueryParams() is not completely thread safe. Can we make a first call in its constructor? This requirement looks funny tho